requirements_dev.txt

```diff
@@ -31,3 +31,5 @@ jax[cpu]
 fastcore
 pyarrow
 metaflow>=2.3.5
+yaspin
+jupyter-repo2docker
```

Comment:
I think you can add `.[local]` which would install all local deps when requirements_dev.txt is installed.

Reply:
Nice, doing that now